packages/definitions/src/integration.ts

```diff
@@ -49,6 +49,12 @@ export const integrationDefs = {
     iconUrl: "https://cdn.jsdelivr.net/gh/homarr-labs/dashboard-icons@master/png/qbittorrent.png",
     category: ["downloadClient", "torrent"],
   },
+  aria2: {
+    name: "Aria2",
+    secretKinds: [["apiKey"]],
+    iconUrl: "https://cdn.jsdelivr.net/gh/homarr-labs/dashboard-icons@master/png/aria2.png",
+    category: ["downloadClient", "torrent", "http(s)"],
```

Comment:
I dislike that the category has brackets in it's name. Just a suggestion: ``multiProtocolDownloadClient``?
@Meierschlumpf / @SeDemal WDYT?

Reply:
updated to use "miscellaneous"